api/src/main/java/org/pmiops/workbench/cloudtasks/TaskQueueService.java

```diff
@@ -87,130 +100,94 @@ public TaskQueueService(
   }
 
   public void groupAndPushRdrWorkspaceTasks(List<Long> workspaceIds) {
-    groupAndPushRdrTasks(workspaceIds, RDR_EXPORT_WORKSPACE_PATH, false);
+    groupAndPushRdrTasks(workspaceIds, RDR_WORKSPACE);
   }
 
-  public void groupAndPushRdrWorkspaceTasks(List<Long> workspaceIds, boolean backfill) {
-    groupAndPushRdrTasks(workspaceIds, RDR_EXPORT_WORKSPACE_PATH, backfill);
+  public void groupAndPushRdrWorkspaceBackfillTasks(List<Long> workspaceIds) {
+    groupAndPushRdrTasks(workspaceIds, withRdrBackfill(RDR_WORKSPACE));
   }
 
   public void groupAndPushRdrResearcherTasks(List<Long> userIds) {
-    groupAndPushRdrTasks(userIds, RDR_EXPORT_RESEARCHER_PATH, false);
+    groupAndPushRdrTasks(userIds, RDR_RESEARCHER);
   }
 
-  public void groupAndPushRdrResearcherTasks(List<Long> userIds, boolean backfill) {
-    groupAndPushRdrTasks(userIds, RDR_EXPORT_RESEARCHER_PATH, backfill);
+  public void groupAndPushRdrResearcherBackfillTasks(List<Long> userIds) {
+    groupAndPushRdrTasks(userIds, withRdrBackfill(RDR_RESEARCHER));
   }
 
-  public void groupAndPushRdrTasks(List<Long> ids, String pathBase, boolean backfill) {
+  public void groupAndPushRdrTasks(List<Long> ids, TaskQueuePair pair) {
```

Comment:
I wonder if the backfill logic could be made more intuitive by adding a backfill parameter here and then calling `withRdrBackfill` if appropriate.

Reply:
Oh ok, given the context, that makes sense. Thanks! :-)